EIPS/eip-5539.md

```diff
@@ -0,0 +1,232 @@
+---
+eip: 5539
+title: Revocation List Registry
+description: Registry of revocation lists for revoking arbitrary data.
+author: Philipp Bolte (@strumswell), Lauritz Leifermann (@lleifermann), Dennis von der Bey (@DennisVonDerBey)
+discussions-to: <URL>
```

Comment:
Required: Please create a thread on Ethereum magicians

Reply:
As far as I can tell my account is still blocked in the forum.

Reply:
Strange... could you send a screenshot?

Reply:
it's fixed now